packages/@webex/internal-plugin-device/src/device.js

```diff
@@ -460,6 +463,20 @@ const Device = WebexPlugin.extend({
         body,
         headers,
       })
+        .catch((error) => {
```

Comment:
suggestion: this could be consolidated a bit by adding the response event to the ```.then``` on line 480  and in the ```.catch``` on line 487 where we send the behavioural metrics

Reply:
it was done like this because the catch on line 487 catches the failed request OR the `processRegistrationSuccess` in the .then on line 480.

We're only concerned with the request for this latency.
@shnaaz explained this to me, I also commented on this in person in the office.

I guess the .then could be consolidated tho.. and register the event right at the beginning of the .then.

Reply:
I have not consolidated the .then because I would like to register the event as soon as the reponse has been received and return and would not like to wait for `processRegistrationSuccess` to process